docs/concepts-index.md

```diff
@@ -0,0 +1,8 @@
+---
+permalink: /concepts-index
+---
+# Concepts index
```

Comment:
delete "index" from the title

Reply:
Since I'm including the list of concept pages in the sidebar, I don't need this anymore.